tests/utilities/fake_discovery_service.py

```diff
@@ -11,26 +11,26 @@ class FakeRegistrationResponse:
 class FakeDiscoveryServiceStub:
     """Fake Registry Service Stub."""
 
-    def RegisterService(self, request):  # noqa N802:inherited method names-autogen baseclass
+    def RegisterService(self, request):  # noqa: N802 - function name should be lowercase
```

Comment:
If lower case function names are the Python standard, why not just rename these? When are we supposed to use lowercase and when aren't we?

Reply:
gRPC codegen chose these names.

Reply:
Oh, duh. Yeah makes sense.